### DISCUSSION TITLE
Azure Data Studio - Fundamentos do SQL Server

### DISCUSSION THREAD

**Discussion by LuizH1359** [balta-io/forum#215] [Tira Dúvidas] [answered]
Estou tentando prosseguir com minha carreira, porém ao iniciar o curso de fundamentos sql, estou tentando começar a aula porém fica dando esse erro, acredito que configurei errado algo do tipo, tem como reiniciar essa configuração, ou mesmo ver como está e se possível alterar também ?
![erro azure](https://user-images.githubusercontent.com/94483864/211409729-fdc9c141-67cc-4791-af98-371b64237828.jpg)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/129

**Reply by LuizH1359** [reply]
O problema dele é diferente do meu, até executei o código indicado no final porém aparece essa mensagem de erro ao executar 
Server=localhost,1433;Database=balta;User ID=sa;Password=1q2w3e4r@#$;Trusted_Connection=False; TrustServerCertificate=True;
aparece está mensagem, não tem como eu limpar e começar esse processo do zero ?
já tentei colocar outra senha, outro usuario e sempre da esse erro
![erro azure dois](https://user-images.githubusercontent.com/94483864/211551093-4f7ef6a3-bae7-4bf4-8488-4c732de75490.jpg)

**Reply by andrebaltieri** [reply]
Bom dia, @LuizH1359, como vai?
O código que você digitou no terminal é sua **Connection String**, não  um comando.

Esta linha é usada dentro do C# para se conectar ao banco de dados.

**Comment by LuizH1359** [comment]
Boa tarde, até consegui abrir e criar o começo, porém voltei depois do
almoço e erro voltou a aparecer, parece que se eu desligar o computador e
voltar a utilizar da erro, ou mesmo sinto que hora que quer funciona hora
que não quer não funciona

A network-related or instance-specific error occurred while establishing a
connection to SQL Server. The server was not found or was not accessible.
Verify that the instance name is correct and that SQL Server is configured
to allow remote connections. (provider: TCP Provider, error: 0 - Nenhuma
conexão pôde ser feita porque a máquina de destino as recusou ativamente.)

Em qua., 11 de jan. de 2023 às 09:08, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Bom dia, @LuizH1359 para resolver este erro:
```
A network-related or instance-specific error occurred while establishing a
connection to SQL Server. The server was not found or was not accessible.
Verify that the instance name is correct and that SQL Server is configured
to allow remote connections. (provider: TCP Provider, error: 0 - Nenhuma
conexão pôde ser feita porque a máquina de destino as recusou ativamente.)
```

Você precisa seguir estes passos aqui, conforme mencionei acima:
👉 https://github.com/balta-io/forum/discussions/129

**Comment by LuizH1359** [comment]
Boa tarde, eu tentei apagar todos os passos que já tinha feito, então
desinstalei tudo e comecei novamente, porém dessa vez quando cheguei na
parte de instalar o .Net estou obtendo esse erro, tentei diretamente pelo
wsl ou mesmo pelo terminal porém ele não está instalando o .net
corretamente, e pelo que pude perceber diz algo sobre essas libs

Em qui., 12 de jan. de 2023 às 10:10, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Boa tarde, @LuizH1359, qual erro está tendo?

**Comment by LuizH1359** [comment]
Boa tarde, eu estava tendo um erro de que quando voltava a entrar no azure
não conectava no banco de dados ficava dando erro, então exclui tudo e
comecei o processo novamente, porém agora quando tento instalar o dotnet
fica dando erro

Em seg., 16 de jan. de 2023 às 15:04, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Boa tarde, @LuizH1359, como vai?
Qual erro?

**Comment by LuizH1359** [comment]
The following packages have unmet dependencies:
 dotnet-runtime-deps-3.1 : Depends: libssl1.0.0 but it is not installable or
                                    libssl1.0.2 but it is not installable or
                                    libssl1.1 but it is not installable
E: Unable to correct problems, you have held broken packages.
***@***.***:/mnt/c/Users/luizh$ dotnet --version
A fatal error occurred. The folder [/usr/share/dotnet/host/fxr] does not
exist

Em ter., 17 de jan. de 2023 às 18:17, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
@LuizH1359 qual sistema operacional está utilizando?

**Comment by LuizH1359** [comment]
Em qua., 18 de jan. de 2023 às 09:54, Andre Baltieri <
***@***.***> escreveu:

> @LuizH1359 <https://github.com/LuizH1359> qual sistema operacional está
> utilizando?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/215#discussioncomment-4717169>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQ3LGBLPCBVU2QWUQB5CELWS7RW3ANCNFSM6AAAAAATV4PG7E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Bom dia, utilizo o sistema Windows , porém como no tutorial instalei o
Ubuntu e o wls

**Reply by andrebaltieri** [reply]
Bom dia, @LuizH1359, como vai?
E qual versão do .NET você está tentando instalar e recebeu o erro acima?

**Comment by LuizH1359** [comment]
Em qui., 19 de jan. de 2023 às 11:21, Andre Baltieri <
***@***.***> escreveu:

> Bom dia, @LuizH1359 <https://github.com/LuizH1359>, como vai?
> E qual versão do .NET você está tentando instalar e recebeu o erro acima?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/215#discussioncomment-4728193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQ3LGGDALSBURYGTVSNPDLWTFEXZANCNFSM6AAAAAATV4PG7E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Estou usando o comando que está no tutorial mesmo

**Comment by LuizH1359** [comment]
Boa tarde, pelo que pude perceber diz que não posso instalar o dotnet
porque não tenho esses arquivos intalados
 libssl1.0.0 but it is not installable or
                                    libssl1.0.2 but it is not installable or
                                    libssl1.1 but it is not installable
E: Unable to correct problems, you have held broken packages.

poderia me ajudar ? Estou realmente precisando sair dessa parte para poder
prosseguir com a carreira

Em qui., 19 de jan. de 2023 às 11:40, Luiz Henrqique Campos Rocha <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Boa tarde, @LuizH1359, como vai?
Como mencionei na resposta anterior, preciso saber qual versão do .NET você está tentando instalar e em qual sistema operacional.

Sem isso não consigo ter ideia do que possa ser.

**Comment by LuizH1359** [comment]
Boa tarde, estou tentando instalar a versão 3.1 do sdk (a que está no
tutorial), e meu sistema operacional é windows porém estou instalando no
wsl que seria linux

Em qua., 25 de jan. de 2023 às 12:24, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Bom dia, @LuizH1359, como vai?

Para instalar o .NET no Windows igual usamos nos cursos, basta executar o EXE que pode ser baixado aqui:
https://dotnet.microsoft.com/en-us/download/dotnet/3.1

Caso queira instalar em uma distribuição Linux, precisa seguir passos adicionais conforme mostrados aqui:
https://learn.microsoft.com/pt-br/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website

No caso do Ubuntu, estes passos são necessários:
https://learn.microsoft.com/pt-br/dotnet/core/install/linux-ubuntu

**Comment by LuizH1359** [comment]
Andre percebi que tive o mesmo erro
https://github.com/balta-io/forum/discussions/27

porém como faço pra adicionar oque você pede na connection string, pois
abri um windowns terminal como administrador e usei os dois comandos porém
não resolveu, mas não sei como adiciono as informações que pediu a
connection string.

Em qui., 26 de jan. de 2023 às 08:53, Andre Baltieri <
***@***.***> escreveu:

**Comment by LuizH1359** [comment]
Boa tarde, consegui resolver aqui viu

Em ter., 31 de jan. de 2023 às 06:26, Luiz Henrqique Campos Rocha <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Perfeito, @LuizH1359 🚀
